set-start.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+var save = process.env.npm_config_save
```

Comment:
is `npm_config_save` documented somewhere? If yes, could you add a comment above with the URL to it? If not, you could mention that `process.env.npm_config_save` is true when running `npm install` with either `-S` or `--save`.

Maybe what would also help is to rename the variable to something more expressive, like `hasNpmSaveFlag`?